packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -63,7 +63,7 @@ const ARB_SEPOLIA_NATIVE_USDC_TOKEN = {
   l2Address: CommonAddress.ArbitrumSepolia.USDC
 }
 
-const TokenListsPanel = memo(function TokenListsPanel() {
```

Comment:
@chrstph-dvx this can't be memo-ed or else the toggle wouldn't work :'(

i have done some refactoring on this branch instead:
#1519